docs/portal/team-members.md

```diff
@@ -37,11 +37,15 @@ To add a team member, complete the following steps:
 1. Log in to your UID2 Portal account.
 1. At the top right, click the down arrow and choose **Manage Team Members**.
 1. Click **Add Team Member**.
-1. Enter the first name, last name, email, and job function.
+1. Enter the first name, last name, email, job function and role.
 1. Click **Save Team Member**.
 
    The team member is added to the list, with Pending status, and an invitation email is sent to the invited person.
 
+:::note
+Team members can be added to more than one participant.
```

Comment:
"can belong" instead of "can be added".  I think here would be helpful even though it is implied to add a sentence that explicitly connects this sentence with the participant switcher - like "if a team member belongs to more than one participant, they will be able to switch between participants as shown in the image"

Reply:
I kept "can be added" because it is under the "Add a Team Member" section, but I've also made changes to use the "belong" terminology and connect it to the switcher.